pyartifactory/objects.py

```diff
@@ -371,7 +371,9 @@ def get(self, name: str) -> Group:
         :return: Found artifactory group
         """
         try:
-            response = self._get(f"api/{self._uri}/{name}")
+            response = self._get(
+                f"api/{self._uri}/{name}", params={"includeUsers": True}
```

Comment:
Apparently, this param appeared on the version 6.13 of Artifactory. What happens for earlier versions ?